certified-connectors/SmartGlobalGovernance/apiDefinition.swagger.json

```diff
@@ -0,0 +1,992 @@
+						"schema": {
+							"type": "object",
+							"properties": {
+								"data": {
+									"type": "array",
+									"items": {
+										"$ref": "#/definitions/DataSubjectRequest"
+									}
+								},
+								"errors": {
+									"$ref": "#/definitions/Errors"
+								},
+								"status": {
+									"$ref": "#/definitions/Status"
+								}
+							}
+						}
+					},
+					"401": {
+						"description": "Authentication failed"
+					}
+				}
+			}
+		},
+		"/request/{request_id}": {
+			"get": {
+				"tags": [
+					"requests"
+				],
+				"summary": "Get a Data Subject Request",
+				"description": "Get a Data Subject Request by its ID",
+				"operationId": "DSR-get-by-id",
+				"parameters": [{
+						"description": "ID of the Data Subject Request that needs to be retrieved",
+						"name": "request_id",
+						"in": "path",
+						"required": true,
+						"type": "integer",
+						"x-ms-summary": "Data Subject Request ID",
+						"x-ms-url-encoding": "single"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "A Data Subject Request",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"data": {
+									"$ref": "#/definitions/DataSubjectRequest"
+								},
+								"errors": {
+									"$ref": "#/definitions/Errors"
+								},
+								"status": {
+									"$ref": "#/definitions/Status"
+								}
+							}
+						}
+					},
+					"401": {
+						"description": "Authentication failed"
+					},
+					"404": {
+						"description": "Data Subject Request ID not found"
+					}
+				}
+			},
+			"put": {
+				"tags": [
+					"requests"
+				],
+				"summary": "Update a Data Subject Request",
+				"description": "Update a Data Subject Request by its ID",
+				"operationId": "DSR-update-by-id",
+				"parameters": [{
+						"description": "ID of the Data Subject Request that needs to be updated",
+						"name": "request_id",
+						"in": "path",
+						"required": true,
+						"type": "integer",
+						"x-ms-summary": "Data Subject Request ID",
+						"x-ms-url-encoding": "single"
+					}, {
+						"name": "body",
+						"in": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/DataSubjectRequest"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "The Data Subject Request was updated successfully"
+					},
+					"401": {
+						"description": "Authentication failed"
+					},
+					"404": {
+						"description": "Data Subject Request ID not found"
+					}
+				}
+			}
+		},
+		"/data-processing": {
+			"post": {
+				"tags": [
+					"data processings"
+				],
+				"summary": "Create a Record of Processing Activity",
+				"description": "Create a new Record of Processing Activity and its properties",
+				"operationId": "ROPA-create",
+				"parameters": [{
+						"name": "body",
+						"in": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/DataProcessing"

```

Comment:
@Philippe-Tivolle As per the swagger submitted in the certification portal, It is `"$ref": "#/definitions/DataSubjectRequest"`

Reply:
There was a mistake there : in that part, we deal with "Data Processing" and not "Data Subject Request".